config.md

```diff
@@ -384,31 +384,82 @@ For POSIX platforms, the configuration structure supports `hooks` for configurin
         * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2008's `environ`][ieee-1003.1-2008-xbd-c8.1].
         * **`timeout`** (int, OPTIONAL) is the number of seconds before aborting the hook.
             If set, `timeout` MUST be greater than zero.
+    * **`create-runtime`** (array of objects, OPTIONAL) is an array of [create-runtime hooks](#create-runtime-hooks).
+        Entries in the array have the same schema as prestart entries.
+    * **`create-container`** (array of objects, OPTIONAL) is an array of [create-container hooks](#create-container-hooks).
+        Entries in the array have the same schema as prestart entries.
+    * **`start-container`** (array of objects, OPTIONAL) is an array of [start-container hooks](#start-container-hooks).
+        Entries in the array have the same schema as prestart entries.
     * **`poststart`** (array of objects, OPTIONAL) is an array of [post-start hooks](#poststart).
-        Entries in the array have the same schema as pre-start entries.
+        Entries in the array have the same schema as prestart entries.
```

Comment:
nit: "prestart" isn't really a word -- write it as `prestart` (the same applies throughout your changes).

Reply:
Done! I ensure the following:
- All references to post-start and post-stop have been renamed to `poststart` and `poststop`
- All references of prestart, create-runtime, ... have been surrounded with backticks.